src/services/ai.ts

```
@@ -0,0 +1,641 @@
+    });
+  }
+
+  getName(): string {
+    return 'Google AI';
+  }
+
+  async validateConnection(): Promise<boolean> {
+    try {
+      await this.client.get('/models');
+      return true;
+    } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)